ibis/expr/api.py

```diff
@@ -191,8 +191,8 @@
 True
 """
 
-
-_ = deferred = Deferred()
+# TODO(kszucs): consider to use `table` variable name instead
+_ = deferred = var("_")
```

Comment:
Since we construct a `var("_")` in `ibis.common.deferred` as well, perhaps just import it from there?

Also regarding the TODO, we already have `ibis.table`, so can't really name this `table`.

Reply:
What I mean under the todo note is:

```py
_ = deferred = var("table")
```

which should be used as 

```py
deferred.resolve(table=table)
```

or

```py
builder.build({"table": table})
```

which could be a more explicit regarding the nature of the `_` placeholder.

Reply:
The point of the nice repr's for deferred expressions is to show them the same as the code the user wrote to generate them. Defining `_ = var("table")` would be non-ideal in that case, since it would then repr as `table` instead of `_`.

Reply:
Removed the comment.

Reply:
Keeping as is.